apps/web/src/views/universalFarms/components/StakeModal.tsx

```diff
@@ -0,0 +1,111 @@
+import styled from 'styled-components'
+import { AtomBox, AutoColumn, Button, Image, Modal, ModalV2, StyledTooltip, Text } from '@pancakeswap/uikit'
+import { useTranslation } from '@pancakeswap/localization'
+import { useTheme } from '@pancakeswap/hooks'
+import { LightCard } from 'components/Card'
+
+const Arrow = styled.div`
+  position: absolute;
+  top: 0px;
+  transform: translate3d(0px, 62px, 0px);
+  right: 4px;
+  &::before {
+    content: '';
+    transform: rotate(45deg);
+    background: var(--colors-backgroundAlt);
+    position: absolute;
+    width: 10px;
+    height: 10px;
+    border-radius: 2px;
+    z-index: -1;
+  }
+`
+
+interface IStakeModalProps {
+  outOfRangeUnstaked: boolean
+  onStake?: (e: React.MouseEvent) => void
+  onUnStake?: (e: React.MouseEvent) => void
+  onDismiss: () => void
+  isOpen: boolean
+}
+
+export const StakeModal: React.FC<React.PropsWithChildren<IStakeModalProps>> = ({
+  outOfRangeUnstaked,
+  onStake,
+  onUnStake,
+  isOpen,
+  onDismiss,
+  children,
+}) => {
+  const { t } = useTranslation()
+  const { isDark } = useTheme()
+  return (
+    <ModalV2 isOpen={isOpen} onDismiss={onDismiss} closeOnOverlayClick>
+      <Modal
+        title={outOfRangeUnstaked ? t('Staking') : t('Unstaking')}
+        width={['100%', '100%', '450px']}
+        maxWidth={['100%', null, '450px']}
+      >
+        <AutoColumn gap="16px">
+          <AtomBox
+            position="relative"
+            style={{
+              minHeight: '96px',
+            }}
+          >
+            <AtomBox
+              style={{
+                width: '100%',
+                position: 'absolute',
+                right: 0,
+                bottom: '-23px',
+                display: 'flex',
+                justifyContent: 'flex-end',
+              }}
+            >
+              <StyledTooltip
+                data-theme={isDark ? 'light' : 'dark'}
+                style={{
+                  maxWidth: '160px',
+                  position: 'relative',
+                }}
+              >
+                {outOfRangeUnstaked ? (
+                  <>
+                    {t('Inactive positions will')}
+                    <b> {t('NOT')} </b>
+                    {t('earn CAKE rewards from farm.')}
+                  </>
+                ) : (
+                  t('You may add or remove liquidity on the position detail page without unstake')
+                )}
+                <Arrow />
+              </StyledTooltip>
+              <Image
+                src="/images/decorations/bulb-bunny.png"
```

Comment:
let's add static file to `assets` repo, as far as possible

Reply:
https://github.com/pancakeswap/assets/pull/93